_posts/en/newsletters/2020-01-08-newsletter.md

```diff
@@ -19,7 +19,13 @@ projects.
 
 ## Action items
 
-*None this week.*
+- Bitcoin Optech is planning to hold a third Schnorr and Taproot
+  seminar workshop in London on 5th February 2020. This will cover
```

Comment:
> on 5th February 2020

Im not familiar with this phrasing. @jnewbery  Maybe its just me?

Reply:
Fine to change it to February 5th 2020 for now, although MDY is a peculiarly American date format. Perhaps we should update the style guide to specify DMY or YMD (perhaps ISO 8601).

See https://en.wikipedia.org/wiki/Date_format_by_country for more details.

Reply:
Sounds good. I used newsletter #78's wording as the tipping factor for me.

> The Optech newsletter will return to its regular Wednesday publication schedule on January 8th.

Style guide entry sounds good either way.

Reply:
5th February 2020 is good; I vote for DMY or YMD / ISO8601 :)

Reply:
When spelling out at least the first three letters of the month, there shouldn't be any ambiguity, so I suggest letting it be author's choice like our [spelling rules](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#spelling).  If using just numbers and separators, my vote is also for ISO 8601.

Reply:
I vote for ISO 8601.